### PR TITLE
YouTubeClone

### PR DESCRIPTION
**Pull request by Yash-Parsana**
### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots

Screenshot (421)

Screenshot (448)

Screenshot (449)

Screenshot (440)

Screenshot (450)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Astrodevil) [inline .vscode/settings.json:1]: Delete this folder, Add your project in cards.json and README

Comment (Yash-Parsana) [reply .vscode/settings.json:1]: which folder ?

**Comment by Yash-Parsana** [comment]
Required changes has been done @Astrodevil

**Comment by Astrodevil** [comment]
Changes are not properly reflected on README and cards.json link should be ("Public/YouTube_Clone/index.html")

**Comment by Yash-Parsana** [comment]
i think now its ok @Astrodevil

**Pull request merged**